backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs

```diff
@@ -66,11 +68,13 @@ public async Task<UserPersonalAndTeamSlotsDto> GetUserPersonalAndTeamSlotsAsync(
             var userSlots = availabilitySlots.Where(x => x.Type == SlotType.Personal).ToList();
             var availabilitySlotsGroupByTeams = availabilitySlots
                 .Where(x => x.Type == SlotType.Team)
-                .GroupBy(x => x.TeamName)
+                .GroupBy(x => x.TeamId)
                 .Select(x =>
                     new AvailabilitySlotsGroupByTeamsDto
                     {
-                        Name = x.Key,
+                        Id = x.Key,
+                        Name = x.First().TeamName,
+                        Image = x.First().TeamLogoPath,
```

Comment:
You can group by all Team or by 3 properties to avoid using x.First()

Reply:
we make select from dto model that doesn't have Team entity
https://prnt.sc/dW6xooMygVJy

Reply:
Ok. than you can use the second option

Reply:
fix by multiple columns. thanks )